e2e/commands/import.e2e.js

```diff
@@ -1571,6 +1571,107 @@ describe('bit import', function () {
     });
   });
 
+  describe('component with shared directory across files and dependencies', () => {
+    /**
```

Comment:
let's also make a similar case where the require in bar foo to is-string is done by bit/ 
in this case I expect the src/bar to gone not only the src. (because there is no link file in utils/is-string)